src/cek/CekResult.js

```diff
@@ -10,5 +11,6 @@
  *   result: Either<{error: string}, string | UplcValue>
  *   cost: Cost
  *   breakdown: CostBreakdown
+ *   diagnostics?: UplcLoggingI  // not needed if always passed into CekMachine
```

Comment:
I think we need a separate type for this as I don't think it makes much sense to pass the `UplcLoggingI` callbacks along with the messages as an evaluation result.